### PR TITLE
Functions for oc_forward_df and oc_reverse_df

### LINKED ISSUE #40: oc_forward and oc_reverse output to data frame
`oc_forward` and `oc_reverse` give output in the form of a list of lists. For a `oc_forward_df` function there needs to be a way to go from the list to a data frame of results. This would build off the `opencage_format`, but possibly find a cleaner way to do it. 🤞I will try to work on this and report back.

### PR DESCRIPTION
**Pull request by jessesadler**
Functions to use data frame with placename column for `oc_forward_df` and latitude and longitude columns for `oc_reverse_df` as input for geocoding and return a data frame. This closes #40.

### CODE REVIEW

File: R/utils.R [hunk 1]
```diff
@@ -17,10 +17,14 @@ oc_parse <- function(req, output) {
...
     if (jsn$total_results == 0) {
-      results <- tibble::tibble(lat = NA_real_, lng = NA_real_)
+      results <- tibble::tibble(lat = NA_real_, lng = NA_real_, formatted = NA)
     } else {
       results <- tibble::as.tibble(jsn$results)
     }
+    if ("request" %in% names(jsn)) {
+      results <- tibble::add_column(results, query = query)
+      results <- dplyr::select(results, query, dplyr::everything())
+    }
```
Comment (dpprdan) [inline R/utils.R]: the `dplyr::select` call is not necessary if you add `.before = 1` to `tibble::add_column`

File: R/forward_geocoding.R [hunk 2]
```diff
@@ -102,7 +102,95 @@ oc_forward <-
...
+           no_annotations = FALSE,
+           no_dedupe = FALSE,
+           no_record = FALSE,
+           abbrv = FALSE) {
+
+    placename <- data[[substitute(placename)]]
+    output <- match.arg(output)
+    add_request <- TRUE # Ensure that query column always exists
+    if (output == "short") {
+      no_annotations <- TRUE
```
Comment (dpprdan) [inline R/forward_geocoding.R:129]: I am still not a big fan of 'silently' overriding parameters, even though it makes total sense to set `no_annotations` to `TRUE` here, after all, the annotations are not going to be in the "short" output. We could set the parameter `no_annotations = ifelse(output ==  "short", TRUE, FALSE)` to make it more transparent. It is a bit wordy, though, so it might not be the best solution. What do you think?

File: R/utils.R (hunk 1 above)
Comment (maelle) [reply R/utils.R]: Wow nice tip
